src/Decomposition.C

```diff
@@ -385,6 +399,22 @@ int OctDecomposition::findSplitters(BoundingBox &universe, CProxy_Reader &reader
     delete msg;
   }
 
+  {
```

Comment:
move splitters.sort() to here and then use the splitters to make the states to countAssignments